logdna/client_test.go

```diff
@@ -0,0 +1,57 @@
+package logdna
+
+import (
+	"errors"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var url = "https://api.logdna.com"
+
+func TestClient_View(t *testing.T) {
+	payload := ViewPayload{Name: "test", Query: "test"}
+	client := Client{ServiceKey: servicekey, HTTPClient: &http.Client{Timeout: 15 * time.Second}}
+	result, err := client.CreateView(url, payload)
+	assert.Equal(t, nil, err)
+	assert.Len(t, result, 10)
+}
+
+func TestClient_Alert(t *testing.T) {
+	var channels []Channel
+	channels = append(channels, Channel{Integration: "pagerduty", Key: "Your PagerDuty API key goes here", TriggerInterval: "15m", TriggerLimit: 20})
+	payload := ViewPayload{Name: "test", Channels: channels}
+	client := Client{ServiceKey: servicekey, HTTPClient: &http.Client{Timeout: 15 * time.Second}}
+	result, err := client.CreateAlert(url, payload)
```

Comment:
following the above comment, `url` here would be the url of your test server

https://golang.org/pkg/net/http/httptest/#Server